Created root folder at construction instead of on first add

ofagbemi opened the pull request:
Fixes #255

dretelny commented:
This doesn't work. I added a member as well as a folder and then added an event and assigned the member. When I refreshed, all of the members and events disappeared and the following error showed up in the console (also, when you are fixing this, can you merge master into this branch and push it?): 

![image](https://cloud.githubusercontent.com/assets/5275384/6444652/d75e273e-c0b3-11e4-925e-e77b87d62b71.png)

ofagbemi commented:
Fixed, really dumb mistake on my part. Should be okay now.

On Tue, Mar 3, 2015 at 1:12 AM, Daniela Retelny <notifications@github.com>
wrote:

> This doesn't work. I added a member as well as a folder and then added an
> event and assigned the member. When I refreshed, all of the members and
> events disappeared and the following error showed up in the console (also,
> when you are fixing this, can you merge master into this branch and push
> it?):
>
> [image: image]
> <https://cloud.githubusercontent.com/assets/5275384/6444652/d75e273e-c0b3-11e4-925e-e77b87d62b71.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/StanfordHCI/foundry/pull/270#issuecomment-76740886>.
>

dretelny commented:
Looks good now! Merging!
